postgres_composite_types/forms.py

```diff
@@ -144,9 +145,11 @@ def clean(self, value):
                 try:
                     cleaned_data[name] = field.clean(value.get(name))
                 except forms.ValidationError as error:
+                    prefix = '%(label)s:' if django.__version__ >= '2.1.0' \
```

Comment:
Avoid unneeded ternary operator.

Also string version compares will fail where semver and string sorting follow different rules. Use `distutils.LooseVersion` or compare against the `django.VERSION` tuple.

e.g. like this Python check:

```
PY36 = sys.version_info >= (3, 6)
```

Reply:
Okay, I have pushed a new commit.
And thanks a lot. I'm learning from you:)

Reply:
Don't forget to remove the ternary operator. Much clearer just to assign and use a variable.

Reply:
Why? I didn't understand.

Reply:
Ternary operators decrease readability, and serve no purpose outside of pure functional code (comprehensions, lambdas, etc.). In fact, for most of its life Python didn't even have a ternary operator.

```
condition = snake > 12

result = badger if condition else mushroom
```

vs

```
if snake > 12
    result = badger
else:
    result = condition
```

The second one is much clearer what `result` is. Especially if we have to write multiple different conditions to set `result`.

Reply:
I don't agree about readability when the condition and result values are very simple. Maybe you have guard against it, simply because you don't use it.

Reply:
See https://pressupinc.com/blog/2015/03/ternary-operators-considered-harmful/

Consider this case for readability:

```
result = badger if snake else mushroom if owl else simoncowell
```

If `snake` is `True` and `owl` is `False` what's the value of `result` ?

Reply:
result is badger because snake is True but I'm against using nested ternary condition. Anyway, I've fix it in a new commit:)